server/utils/premises/index.test.ts

```diff
@@ -240,6 +241,50 @@ describe('premisesUtils', () => {
     })
   })
 
+  describe('groupCas1SummaryPremisesSelectOptions', () => {
+    const area1Premises = cas1PremisesSummaryFactory.buildList(2, { apArea: { id: 'a1', name: 'Area 1' } })
+    const area2Premises = cas1PremisesSummaryFactory.buildList(2, { apArea: { id: 'a2', name: 'Area 2' } })
+    const premises = [...area1Premises, ...area2Premises]
+
+    it('should group premises by region', () => {
+      expect(groupCas1SummaryPremisesSelectOptions(premises, { premisesId: area1Premises[1].id })).toEqual([
```

Comment:
Probably no need to test the selected option here -- test case seems to be basic render.

Reply:
It was a copy'n'paste from existing code but I think it's valid. First test case shows selection of the default selection fieldname while second shows an explicit one.

Reply:
The context could be empty, so I'd expect to see a test for:
```suggestion
      expect(groupCas1SummaryPremisesSelectOptions(premises, {})).toEqual([
```

Reply:
That's covered off in the first test - which is testing for the grouping. I thought that specifically testing for the selected flags, it wouldn't need to check the entire list content.